nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/TestRetentionRemove.java

```diff
@@ -268,6 +292,7 @@ public void setUp() {
         addPermission(session.getRootDocument(), JOHN, REMOVE_CHILDREN);
         folder = session.createDocumentModel("/", "myFolder", "Folder");
         folder = session.createDocument(folder);
+        blockPermissionInheritance(folder);
```

Comment:
why?

Reply:
to verify that Administrator does not get its permissions from the ACP, but from the fact that it's a admin user